DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: tinkr
 Title: Cast '(R)Markdown' Files to 'XML' and Back Again
-Version: 0.2.0.9000
```

Comment:
I still think you might enjoy {fledge} since you're so good at incrementing version numbers!